package.json

```diff
@@ -21,6 +21,7 @@
     "vite": "^5.4.1"
   },
   "dependencies": {
-    "kaplay": "^3001.0.0-beta.1"
+    "kaplay": "^3001.0.0-beta.1",
+    "project": "file:"
```

Comment:
can you please revert the changes here in the package.json file?

Reply:
I apologize...but I don't have any idea..what it means..to reverte the change?

Reply:
in your case, just remove 
```json
"project": "file:"
```
and run npm install afterwards which should fix the package-lock.json file as well
@MIhnatco

Reply:
Thanks